packages/lib-classifier/src/store/subjects/Subject/Subject.spec.js

```diff
@@ -57,7 +57,6 @@ describe('Model > Subject', function () {
         get: sinon.stub().callsFake(() => Promise.resolve({ body: { tutorials: [] } }))
       }
       rootStore = mockStore({ workflow: workflowSnapshot, client })
-      rootStore.subjects.reset()
       client.caesar.request.resetHistory()
       await rootStore.subjects.populateQueue()
```

Comment:
I think this can be changed to a promise that waits for the subject queue to be ready, using `when()`.

Reply:
Here's an example from the Classifier tests. You can probably adapt it for this test.
https://github.com/zooniverse/front-end-monorepo/blob/1cb860105e320dab4e18b889b800d44416c9feef/packages/lib-classifier/src/components/Classifier/Classifier.workflows.spec.js#L130